e2e/QUICKSTART.md

```diff
@@ -0,0 +1,243 @@
+}
+
+func InitializeScenario(ctx *godog.ScenarioContext) {
+        ctx.Step(`^I eat (\d+)$`, iEat)
+        ctx.Step(`^there are (\d+) godogs$`, thereAreGodogs)
+        ctx.Step(`^there should be (\d+) remaining$`, thereShouldBeRemaining)
+}
+
+testing: warning: no tests to run
+PASS
+ok      github.com/elastic/e2e-testing/e2e/_suites/foo  1.445s
+```
+
+Copy the content of the outputted `InitializeScenario` method, _the ctx.Step invocations_, into your `foo_test.go`, within the `InitializeFooScenario` struct - `vim foo_test.go`
+``` go
+func InitializeFooScenario(ctx *godog.ScenarioContext) {
+    // ...
+    ctx.Step(`^I eat (\d+)$`, iEat)
+    ctx.Step(`^there are (\d+) godogs$`, thereAreGodogs)
+    ctx.Step(`^there should be (\d+) remaining$`, thereShouldBeRemaining)
+    // ...
+}
+```
+
+Finally copy the outputted implementation methods in the `foo_test.go` file.
+
+Run godog again - `go test -v --godog.format=pretty`
+
+You should now see that the scenario is pending with one step pending and two steps skipped:
+```
+Feature: Foo
+  In order to be happy
+  As a hungry gopher
+  I need to be able to eat godogs
+
+  Scenario: Eat 5 out of 12          # features/foo.feature:7
+    Given there are 12 godogs        # foo_test.go:112 -> github.com/elastic/e2e-testing/e2e/_suites/foo.thereAreGodogs
+      TODO: write pending definition
+    When I eat 5                     # foo_test.go:108 -> github.com/elastic/e2e-testing/e2e/_suites/foo.iEat
+    Then there should be 7 remaining # foo_test.go:116 -> github.com/elastic/e2e-testing/e2e/_suites/foo.thereShouldBeRemaining
+
+1 scenarios (1 pending)
+3 steps (1 pending, 2 skipped)
+1.025856125s
+testing: warning: no tests to run
+PASS
+ok      github.com/elastic/e2e-testing/e2e/_suites/foo  1.331s
+```
+
+You may change **return godog.ErrPending** to **return nil** in the three step definitions and the scenario will pass successfully.
+
+Also, you may omit error return if your step does not fail.
+
+```go
+func iEat(arg1 int) {
+	// Eat arg1.
+}
+```
+
+### Step 5 - Add some logic to the step definitions
+
+At this time we recommend moving all the implementation methods to the `FooTestSuite` struct, so that we can pass state and logic between a scenario's steps with ease.
+
+
+```go
+func InitializeFooScenario(ctx *godog.ScenarioContext) {
+    // ...
+    ctx.Step(`^I eat (\d+)$`, testSuite.iEat)
+    // ...
+}
+
+func (fts *FooTestSuite) iEat(arg1 int) {
+	// Eat arg1.
+}
+```
+
+### Step 6 - Add your custom logic to the step definitions
+
+Now that you created your first scenario, you can continue with the next steps: working on an existing test suite, but it's probably a good idea to check out our brief introduction to `Test specification with Gherkin` below. It will provide you the bare minimal concepts about Gherkin clauses and BDD.
+
+## Test specification with Gherkin
+
+All the Gherkin (Cucumber) specifications are written in `.feature` files. The anatomy of a feature file is:
```

Comment:
It's a bit unclear from reading this section how the Gherkin specs related to the godog explanation above. Do they work together in tandem? If so, how?